css-fonts/styles.css

```diff
@@ -0,0 +1,15 @@
+h1 {
+  font-family: 'Cabin', sans-serif;
+  font-family: 'Source Serif 4', serif;
+}
+
+h2 {
+  font-family: 'Cabin', sans-serif;
+  font-family: 'Source Serif 4', serif;
+}
+
+body {
+  font-family: 'Cabin', sans-serif;
+  font-family: 'Source Serif 4', serif;
+  line-height: 1.6rem;
+}
```

Comment:
Each selector should only have 1 `font-family` property. You're technically giving all of these the same font because your 2nd `font-family` is overriding the first one.